docker/development/pnpm/Dockerfile

```
@@ -7,4 +7,5 @@ RUN npm i -g corepack@latest
 
 RUN corepack enable
 RUN corepack install -g pnpm@10.26.0
-WORKDIR /var/www
\ No newline at end of file
+RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)